File: app/src/test/java/com/robotnec/budget/BaseRobolectricTest.java
package com.robotnec.budget;

import android.os.Build;

import com.robotnec.budget.BuildConfig;

import org.junit.runner.RunWith;
import org.robolectric.RobolectricTestRunner;
import org.robolectric.annotation.Config;

@RunWith(RobolectricTestRunner.class)
@Config(constants = BuildConfig.class, sdk = Build.VERSION_CODES.M)
public abstract class BaseRobolectricTest {
}

File: app/src/test/java/com/robotnec/budget/BaseRobolectricTest.kt
package com.robotnec.budget

import android.os.Build
import org.junit.runner.RunWith
import org.robolectric.RobolectricTestRunner
import org.robolectric.annotation.Config

@RunWith(RobolectricTestRunner::class)
@Config(constants = BuildConfig::class, sdk = intArrayOf(Build.VERSION_CODES.M))
abstract class BaseRobolectricTest